pkg/segment/query/metricsquery.go

```diff
@@ -425,6 +428,16 @@ func applyMetricsOperatorOnSegments(mQuery *structs.MetricsQuery, allSearchReqes
 			if mQuery.IsQueryCancelled() {
 				return
 			}
+
+			if mSeg.QueryType == structs.UNROTATED_METRICS_SEARCH {
```

Comment:
this should be `unrotated_block_metrics_search`, because we can have rotated blocks on disk, but the segment may not be rotated. currently there is no difference between rotated_block and rotated_segment. But in future we might. So we should be explicit about that we are searching unrotated_block